src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/profiler/model/ProfilerContents.java

```diff
@@ -45,4 +45,12 @@ public final native boolean getCreateProfile() /*-{
    public final native String getName() /*-{
       return this.name;
    }-*/;
+   
+   public final native String getHtmlLocalPath() /*-{
+      return this.htmlLocalPath;
+   }-*/;
+   
+   public final native boolean isUserSaved() /*-{
```

Comment:
Sort of, fix was not complete, now it is. This new property tracks the saved state of the document to allow users to see the "save" command enabled only once while their doc has never ben saved. Once it's saved, it can only ve saved again with the "save as" command. This matches the behavior of other docs in the IDE.